lib/reporting/identity_verification_report.rb

```diff
@@ -19,6 +19,7 @@ class IdentityVerificationReport
     module Events
       IDV_DOC_AUTH_WELCOME = 'IdV: doc auth welcome visited'
       IDV_DOC_AUTH_GETTING_STARTED = 'IdV: doc auth getting_started visited'
+      IDV_DOC_AUTH_WELCOME_SUBMITTED = 'IdV: doc auth welcome submitted'
```

Comment:
let's put the welcome steps next to each other
```suggestion
      IDV_DOC_AUTH_WELCOME = 'IdV: doc auth welcome visited'
      IDV_DOC_AUTH_WELCOME_SUBMITTED = 'IdV: doc auth welcome submitted'
      IDV_DOC_AUTH_GETTING_STARTED = 'IdV: doc auth getting_started visited'
```